lib/hooks.js

```diff
@@ -90,7 +90,10 @@ const Hooks = {
 
   async runHooks(hooks, ...hookArgs) {
     if (!hooks) throw new Error('runHooks requires at least 1 argument');
-
+    // Skip hooks if we are only getting raw SQL
+    if (hookArgs[0] && typeof hookArgs[0] === 'object' && hookArgs[0].getRawSql) {
```

Comment:
Hooks can also modify the resulting query, such as `beforeFind`. I think it's best if we extend the `hookTypes` object, and add a new option to each hook that we want to run, when getting the raw SQL query